vllm/spec_decode/multi_step_worker.py

```diff
@@ -237,6 +239,13 @@ def _append_new_tokens(
 
                 token_id = seq_output.output_token
                 token_logprob = seq_output.logprobs[token_id]
```

Comment:
Changing token_logprob here is challenging because the log probability of the bonus token is required. I'm unsure if this change is actually necessary at this point.

Reply:
Will we be using this logprob at all since we will be filtering out these sequences in the final output right? In that case we can add a note saying that this is a fake/unused logprob?

Reply:
yeah, I think the logprobs of the sequence without the bonus token will likely not be used. I will check this precisely and add a note about it.